tests/tools/pytest_fixtures/test_configuration.py

```diff
@@ -1,24 +1,22 @@
 """Test the pytest fixtures."""
 
-import tempfile
 
-
-def test_aiida_config():
+def test_aiida_config(tmp_path):
     """Test that ``aiida_config`` fixture is loaded by default and creates a config instance in temp directory."""
     from aiida.manage.configuration import get_config
     from aiida.manage.configuration.config import Config
 
     config = get_config()
     assert isinstance(config, Config)
-    assert config.dirpath.startswith(tempfile.gettempdir())
+    assert config.dirpath.startswith(str(tmp_path.parent))
```

Comment:
Since we are using the factory fixture in the `aiida_config` fixture, I think the following may make more sense
```suggestion
    assert config.dirpath.startswith(tmp_path_factory.getbasetemp())
```